### DISCUSSION TITLE
How to hide online status when using nchat?

### DISCUSSION THREAD

**Discussion by kurokirasama** [d99kris/nchat#197] [Q&A]
is it possible to change to "not online" status?

**Comment by elitaylor** [comment]
yes, you can change it. Give Privacy setting a shot in the app and you can change the online status there.

**Comment by d99kris** [comment]
You can set [online_status_share](https://github.com/d99kris/nchat#online_status_share) to `0` in your `~/.nchat/ui.conf` (while nchat is not running) and then start nchat again. This should prevent nchat from announcing you're online. 

You can also consider setting [typing_status_share](https://github.com/d99kris/nchat#typing_status_share) to `0` to prevent sharing "typing" status.

**Reply by kurokirasama** [reply]
thanks a lot!
